Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
Running CRIU on with an Arch Linux host currently requires a custom kernel because the default kernel build options don't support checkpoint/restore.

But building this branch doesn't work with the default Arch kernel either.  The  `uname -r` command returns `4.2.5-1-ARCH` (even inside a Ubuntu container) and when I run `make DOCKER_EXPERIMENTAL=1 build` on an Arch Linux system with the default kernel I get an error:

```
E: Unable to locate package linux-image-4.2.5-1-ARCH
E: Couldn't find any package by regex 'linux-image-4.2.5-1-ARCH'
````
I imagine that a similar problem could occur for other non-Ubuntu hosts too.  I guess it would depend on if the kernel naming convention happens to match one used by Ubuntu (I don't have any other distros to test though).